### DISCUSSION TITLE
Wii Sensor Bar

### DISCUSSION THREAD

**Discussion by trident0005** [mholtkamp/octave#15] [Ideas]
Hi, I'm making a game with your engine, and would like to suggest wii sensor bar support for navigating menus.

**Comment by mholtkamp** [comment]
If you mean using the Wiimote to point and select, it should be possible already. You can get the pointer wiimote pointer screen position by using `local x,y = Input.GetPointerPosition()` and you can draw a cursor by adding a Quad to your scene and setting its position like `quad:SetPosition(x,y)` 

Let me know if you have troubles getting it working or if I misunderstood what you were suggesting.
